product/carbon-home/conf/identity/profile-mapping.yaml

```diff
@@ -149,3 +149,21 @@ profiles:
    readonly: false
    regex: "*"
    dataType: "email"
+ -
+  profileName: "user-identifier"
```

Comment:
user list table contains a username column. In C5 there's no one fixed claim for username. Admin can define this profile to define one or more unique claims in the system. And configure what value from them to show as the Username. Should be an admin profile.